decidim-core/app/functions/decidim/core/needs_filter_and_order.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Core
+    module NeedsFilterAndOrder
```

Comment:
Let's be explicit with the name so it will be easier to associate the file name witht he API.
```suggestion
    module NeedsApiFilterAndOrder
```